setup.py

```diff
@@ -16,7 +16,8 @@
     LIB_PACKAGE = ['nbeats_keras']
     INSTALL_REQUIRES.extend([
         'keras',
-        'tensorflow'
+        'tensorflow',
+        'wfdb == 2.2.1'
```

Comment:
wfdb should not be part of setup.py because everyone that will want to use n-beats will have wfdb packaged with it!

Reply:
Updated